.github/workflows/release_cdf-typescript.yml

```diff
@@ -26,7 +30,7 @@ jobs:
       - name: Run Tests
         run: npm run test
       - name: Generate Docs
-        run: npm run doc:*
+        run: npm run doc:html
```

Comment:
Not sure exactly how you were thinking about combining this with the guides?

I'd like to try spitting this out as MD and then using the mkdocs to build into a single site.

Reply:
Good question: we're shipping docs in html in each npm version when releasing. They go into `build/docs` dir therefore not persisted in repo itself.
We definitely should generate another set of docs for doc site for sure, and it may reside in a different workflow.

Reply:
If you make tit output just md into the docs folder mkdocs will handle the html build and publish.